.ci/ci_job_flags.sh

```diff
@@ -162,26 +162,20 @@ case "${CI_JOB}" in
 	export KATA_HYPERVISOR="cloud-hypervisor"
 	export KUBERNETES="yes"
 	;;
-"CLOUD-HYPERVISOR-K8S-CONTAINERD")
+"CLOUD-HYPERVISOR-K8S-CONTAINERD"|"CLOUD-HYPERVISOR-K8S-CONTAINERD-DEVMAPPER"|"EXTERNAL_CLOUD_HYPERVISOR")
```

Comment:
Hey @GabyCT!

I'm on the fence on this one.  While it's a nice refactoring grouping those, I really want to avoid getting to the same situation as we got into the CCv0 branch, which actually make it hard to read, take a look: https://github.com/kata-containers/tests/blob/93f44eb6061364ab8d802dc5941a2dcd21cfd432/.ci/ci_job_flags.sh#L108 and https://github.com/kata-containers/tests/blob/93f44eb6061364ab8d802dc5941a2dcd21cfd432/.ci/ci_job_flags.sh#L151